tests/test_sensor_download.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+
+from tests import test_authorization as Authorization
+from falconpy import sensor_download as FalconSensorDownload
+
+sys.path.append(os.path.abspath('src'))
+AllowedResponses = [200, 429]  # Adding rate-limiting as an allowed response for now
+appId = "pytest-sensor_download-unit-test"
+auth = Authorization.TestAuthorization()
+auth.serviceAuth()
+
+sensor_download_client = FalconSensorDownload.Sensor_Download(access_token=auth.token)
+
+class TestSensorDownload():
+    
+    @staticmethod
+    def _get_cid():
+        resp = sensor_download_client.GetSensorInstallersCCIDByQuery()
+        return True if resp["status_code"] in AllowedResponses else False
+
+    @staticmethod
+    def _get_multiple_shas():
+        params = {"filter": 'platform:"windows"', "sort": "release_date|desc"}
+        shas = sensor_download_client.GetSensorInstallersByQuery(params)["body"].get("resources", ["b7dbe32495c6e23ca2e04784b98d5a1770163b9f42524cafc062292b533f03c9"])
+        return shas
```

Comment:
```suggestion
    def _get_multiple_shas():
        params = {"filter": 'platform:"windows"', "sort": "release_date|desc"}
        shas = sensor_download_client.GetSensorInstallersByQuery(params)["body"].get("resources", [""])
        return shas
```

Reply:
We need to come up with a way to retrieve this value dynamically, or we'll be stuck updating the unit test as the sha changes.

Reply:
if the API returns a 429, we could just retry but dont like doing that in tests typically. that what was causing issues when i was testing, was getting constant 429s so just hard coded one.

I was thinking of reworking tests to utilize  pytest-httpserver but project for another weekend I think. Would need way to get swagger docs into it